models/photos.js

```diff
@@ -22,6 +22,9 @@ const Photos = sql.define("photo", {
 		type: Sequelize.STRING,
 		notNull: true,
 	},
+	description: {
+		type: Sequelize.STRING(500),
+		}
```

Comment:
The problem had the following requirement:

> A user should not be able to:
> * Write a description longer than 150 characters

I think you want this to be 150, not 500!